packages/tracking/src/test/trackEvent.ts

```diff
@@ -34,23 +34,23 @@ describe('trackEvent', () => {
     config.trackingAllowed = true;
     config.trackingEnabled = true;
 
-    gtag.mockImplementationOnce((type, eventName, eventData) => {
-      eventData.event_callback();
+    jest.mocked(gtag).mockImplementationOnce((_type, _eventName, eventData) => {
+      (eventData as ControlParams).event_callback?.();
     });
 
-    await trackEvent('name', { foo: 'abc', bar: 'def', baz: 'ghi' });
+    await trackEvent('name', { name: 'abc', value: 'def', event_label: 'ghi' });
     expect(gtag).toHaveBeenCalledWith('event', 'name', {
-      event_callback: expect.any(Function),
-      foo: 'abc',
-      bar: 'def',
-      baz: 'ghi',
+      event_callback: expect.any(Function) as () => void,
```

Comment:
Hmm, this is a bit weird. This feels wrong with that cast? Though I can't see how it could be any cleaner than that though - you can't type `expect.any` correctly, can you? Maybe with an overload for every constructor type? That's besides the point though, not much to do here.

Reply:
Looks like they're completely unnecessary